recipes/xarray-einstats/meta.yaml

```
@@ -0,0 +1,46 @@
+# Generated using grayskull; edited by @ericmjl.
+{% set name = "xarray-einstats" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xarray-einstats-{{ version }}.tar.gz
+  sha256: 5c4e00d09172daf665cbd5e533a897896b84f37feb5cab0fd75829477fbcda7d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip >=21.3
+    - python >=3.7
+    - flit
+  run:
+    - numpy >=1.21
+    - python >=3.7
+    - scipy
+    - xarray >=0.20
+
+test:
+  imports:
+    - xarray_einstats
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/xarray-einstats/
```

```suggestion
  home: https://github.com/arviz-devs/xarray-einstats
```

I think the arviz feedstock takes the license from github directly using the link provided here: https://github.com/conda-forge/arviz-feedstock/blob/main/recipe/meta.yaml#L41

---

I can try to add the license to pip package too but I'd need an example of how to do it (I tried to copy arviz and saw arviz doesn't include it in the pip package)

---

I think I found an example to copy, @OriolAbril! 

`pyjanitor` uses a MANIFEST.in file to declare which other files need to be packaged together and distributed. 

When I looked up other references, I stumbled upon the rules by which Python package files are included: https://packaging.python.org/en/latest/guides/using-manifest-in/. Perhaps that could help as well.